### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by kunicmarko20**
Not sure if we even need an end date.

### CODE REVIEW

File: project/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2010-2016 Thomas Rabaix
+Copyright (c) 2010-2018 Thomas Rabaix
```
Comment (greg0ire) [inline project/LICENSE]: I think I read somewhere that you can just put "…2010 Thomas Rabaix" and be done with it forever

Comment (jordisala1991) [reply project/LICENSE]: o use twig to generate a date

Comment (kunicmarko20) [reply project/LICENSE]: https://github.com/symfony/form/commit/973a310098c590abe3e35a90ed2a7389f26158b3

was overridden with:

https://github.com/symfony/form/commit/0d89adcacf5be46c572d074302a0e0aa52e25fee

So I guess Symfony updates it every year, should we?

Comment (kunicmarko20) [reply project/LICENSE]: @greg0ire I saw that too but never find something that confirms it.

**Comment by kunicmarko20** [comment]
So, 2018 or dropping end date? @sonata-project/contributors

**Comment by greg0ire** [comment]
For me, dropping end date

**Comment by kunicmarko20** [comment]
As @core23 also agrees and I don't really care, lets drop it then :smile:

**Comment by kunicmarko20** [comment]
cc @Soullivaneuh

**Pull request merged**